index.js

```diff
@@ -103,16 +121,19 @@ exports.single_recipient = function (next, connection) {
 
   transaction.results.add(this, { fail: 'single_recipient', emit: true })
 
-  if (!this.cfg.reject.single_recipient) return next()
+  if (this.cfg.reject.single_recipient) {
+    next(DENY, 'this bounce message has too many recipients')
+  }
 
-  next(DENY, 'this bounce message does not have 1 recipient')
+  return next()
```

Comment:
```suggestion
  next()
```

Avoid a useless use of `return` here.